### DISCUSSION TITLE
Welcome to nightTab Discussions!

### DISCUSSION THREAD

**Discussion by zombieFox** [zombieFox/nightTab#274] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Share nightTab setups and screenshots 📷 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sky-361** [comment]
Hey, I am Sky. I am just here to suggest different ideas in order to make the user experience better.

**Comment by Sky-361** [comment]


**Reply by zombieFox** [reply]
Hi Sky, thanks for introducing yourself and providing some ideas. This discussions is a place to post introductions. Happy to discuss the ideas you listed -- please open up a [new discussion](https://github.com/zombieFox/nightTab/discussions/new) from the main page. Also do provide lots of details, images or flows to help describe how you imagine new features might function.

**Reply by Sky-361** [reply]
Oh, ok. Thanks.

**Comment by whitefox-noot** [comment]
how to get a https and a mp4 at the same time as i find that when i download a video i only have file mp4 and not https mp4, how do i fix that. Pls help
